src/ruby/yast/yast.rb

```diff
@@ -197,6 +197,37 @@ def self.#{sname}= (value)
     base.const_set(modules.last, m) unless base.constants.include?(modules.last.to_sym)
   end
 
+  def self.stftime(time, format)
```

Comment:
I think this is not correct place, it should go to Builtins class and not whole yast namespace, otherwise it confuse anyone as they think it is ruby version of strftime and not our one.